tests/test_faster_rcnn.py

```diff
@@ -0,0 +1,377 @@
+import argparse
+import os
+from unittest.mock import MagicMock, patch
+
+import numpy as np
+import pytest
+import torch
+import torchvision
+from PIL import Image
+from tensorboardX import SummaryWriter
+from yacs.config import CfgNode as CN
+
+from datasetinsights.data.dummy.dummy_object_detection import DummyDetection2D
+from datasetinsights.estimators import Estimator
+from datasetinsights.estimators.faster_rcnn import (
+    collate_fn,
+    create_dataloader,
+    create_dryrun_dataset,
+    create_optimizer_lrs,
+    create_sampler,
+    download_data,
+    get_transform,
+    load_data,
+)
+from datasetinsights.storage.checkpoint import create_checkpointer
+
+TRAIN = "train"
```

Comment:
Why do you need to redefine these constants? Can you read from estimators/faster_rcnn.py?

Reply:
Fixed